plot/indicator/bollingerbands.go

```diff
@@ -0,0 +1,72 @@
+package indicator
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/rodrigo-brito/ninjabot/model"
+	"github.com/rodrigo-brito/ninjabot/plot"
+
+	"github.com/markcheno/go-talib"
+)
+
+func BollingerBands(period, stdDeviation int, upDnBandColor, midBandColor string) plot.Indicator {
+	return &bollingerBands{
+		Period:        period,
+		StdDeviation:  stdDeviation,
+		UpDnBandColor: upDnBandColor,
+		MidBandColor:  midBandColor,
+	}
+}
+
+type bollingerBands struct {
+	Period        int
+	StdDeviation  int
+	UpDnBandColor string
+	MidBandColor  string
+	UpperBand     model.Series
+	MiddleBand    model.Series
+	LowerBand     model.Series
+	Time          []time.Time
+}
+
+func (bb bollingerBands) Name() string {
+	return fmt.Sprintf("BB %d close %d", bb.Period, bb.StdDeviation)
+}
+
+func (bb bollingerBands) Overlay() bool {
+	return true
+}
+
+func (bb *bollingerBands) Load(dataframe *model.Dataframe) {
+	if len(dataframe.Time) < bb.Period {
+		return
+	}
+
+	deviation := float64(bb.StdDeviation)
+	bb.UpperBand, bb.MiddleBand, bb.LowerBand = talib.BBands(dataframe.Close, bb.Period, deviation, deviation, talib.EMA)
```

Comment:
You need to limit the bands, similar to `bb.Time`.
```suggestion
	upper, mid, lower := talib.BBands(dataframe.Close, bb.Period, deviation, deviation, talib.EMA)
	bb.UpperBand, bb.MiddleBand, bb.LowerBand = upper[bb.Period:], mid[bb.Period:], lower[bb.Period:]
```